### PR TITLE
Fix orgList screen refreshing entire data on load more orgs

### LINKED ISSUE #1359: Organization List screen for SuperAdmin/Admin is refreshing completely while loading more organizations
**Describe the bug**
Organization List screen for SuperAdmin/Admin is refreshing completely while loading more organizations

**To Reproduce**

1. Go to organization list screen
2. Set the network throttling to 2G
3. Scroll down further
4. Whole screen is getting reloaded

**Expected behavior**
The already loaded organizations should not reload

**Actual behavior**
Whole screen loading animation is played

**Screenshots**

https://github.com/PalisadoesFoundation/talawa-admin/assets/76212518/9ae0afd2-c839-4146-8121-0da0d5703cc5

**Additional details**

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by NayOoLwin5**
**What kind of change does this PR introduce?**

This PR fixes the issue when loading more orgs in organization list data, the entire data refreshes. With this PR, instead of refreshing entire data, it only refreshes the newly fetched data through pagination.

**Issue Number:**

Fixes #1359 

**Did you add tests for your changes?**

**Snapshots/Videos:**

https://github.com/PalisadoesFoundation/talawa-admin/assets/52243229/f9127e5e-1135-4905-a5e8-196c15b0f56b

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

**Comment by Cioppolo14** [comment]
@NayOoLwin5 Please fix the failing introspection test.

**Comment by Cioppolo14** [comment]
@NayOoLwin5 We are working on a fix for the UpdateUserInput, but for now can you please update your schema file?

**Comment by Cioppolo14** [comment]
@noman2002 @anwersayeed Can you review this PR?

File: src/GraphQl/Mutations/mutations.ts [hunk 1]
```diff
@@ -84,21 +84,13 @@ export const ADDRESS_DETAILS_FRAGMENT = gql`
...
   ) {
     updateUserProfile(
-      data: {
-        firstName: $firstName
-        lastName: $lastName
-        email: $email
-        id: $id
-        applangcode: $applangcode
-      }
+      data: { firstName: $firstName, lastName: $lastName, email: $email }
```
Comment (SiddheshKukade) [inline src/GraphQl/Mutations/mutations.ts:93]: why did you removed applangcode, id from here ?

**Review by SiddheshKukade — CHANGES_REQUESTED** [review]

File: src/GraphQl/Mutations/mutations.ts (hunk 1 above)
Comment (NayOoLwin5) [reply src/GraphQl/Mutations/mutations.ts:93]: Hi @SiddheshKukade, If I don't remove those fields from the schema, introspection test failed. Which is why I was told to remove them for now until [#1762](https://github.com/PalisadoesFoundation/talawa-api/pull/1762) merged in talawa-api side. The PRs of talawa-admin cannot be merged without removing those fields until the talawa-api PR is merged.

**Comment by NayOoLwin5** [comment]
@noman2002 @anwersayeed @palisadoes I have restored the schema code that I removed earlier having noticed the information that it is not right way to pass the known introspection test error. I believe the failed test can be skipped. If not, please let me know how I can proceed with it.

**Comment by noman2002** [comment]
@palisadoes Is the introspection check supposed to fail?

**Comment by noman2002** [comment]
Changes look fine to me.

**Comment by palisadoes** [comment]
Merging. I have created an issue to fix introspection.
The API is slightly ahead of the Admin feature set - @Cioppolo14  FYI

**Review by palisadoes — APPROVED** [review]

**Pull request merged**